pwndbg/commands/context.py

```diff
@@ -278,21 +279,23 @@ def context_backtrace(frame_count=10, with_banner=True):
     return result
 
 
-def context_args():
+def context_args(with_banner=True):
     result = []
 
-    ##################################################
-    # DISABLED FOR NOW, I LIKE INLINE DISPLAY BETTER
-    ##################################################
-    # # For call instructions, attempt to resolve the target and
-    # # determine the number of arguments.
-    # for arg, value in pwndbg.arguments.arguments(pwndbg.disasm.one()):
-    #     code   = False if arg.type == 'char' else True
-    #     pretty = pwndbg.chain.format(value, code=code)
-    #     result.append('%-10s %s' % (arg.name+':', pretty))
-    # if not result:
-    #         return []
-    # result.insert(0, pwndbg.ui.banner("arguments"))
+    # For call instructions, attempt to resolve the target and
+    # determine the number of arguments.
+    one = pwndbg.disasm.one()
+    for arg, value in pwndbg.arguments.get(one):
+        code   = False if arg.type == 'char' else True
+        pretty = pwndbg.chain.format(value, code=code)
+        result.append('%8s%-10s %s' % ('', N.argument(arg.name) + ':', pretty))
+
+    if not result:
+            return []
```

Comment:
Something wrong with indentation here.

Reply:
This is correct, point is to return without banner and results if there are no arguments for the current instruction as ints not a call